centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -24579,3 +24579,6 @@ msgstr "alerte"
 #: unknown
 msgid "will display resources with host name starting by'"
 msgstr "affichera les ressources avec un nom d'hôte commençant par"
+
+msgid "Too many elements to be displayed"
+msgstr "Trop d'éléments à afficher" 
```

Comment:
Maybe change the text to "There is too many etc..."

Reply:
It was documentation team that suggested this text.